pages/common/tokei.md

```diff
@@ -0,0 +1,19 @@
+# tokei
+
+> A program that prints out statistics about your code.
```

Comment:
Hey @AnimiVulpis - I deliberately mentioned "one's" because we avoid using 2nd person in our pages. Kindly change it to "one's" or feel free to use something else that doesn't mention "your".